dali/ft/filecopy.cpp

```diff
@@ -2566,6 +2568,12 @@ void FileSprayer::performTransfer()
         numSlavesCompleted++;
     }
 
+#ifdef RUN_SLAVES_ON_THREADS
```

Comment:
Any reason not to join transferSlaves unconditionally even if  RUN_SLAVES_ON_THREADS is false?

Reply:
Because they haven't been started on threads.  It is more logically consistent to only join if they have been started.